zstacklib/zstacklib/test/test_parse_lspci.py

```
@@ -0,0 +1,331 @@
+NUMANode:       0
+IOMMUGroup:     80'''
+
+
+class Test(unittest.TestCase):
+    results = pci._parse_lspci(value_output, id_output)  # type: list[dict]
+
+    assert len(results) == 9
+
+    for res in results:
+        if res['Slot'] == "0000:00:00.0":
+            assert '0000:00:00.0' == res['Slot']
```

Ruff (0.11.9)

Yoda condition detected

Rewrite as `res['Slot'] == '0000:00:00.0'`

(SIM300)